### ISSUE TITLE
Cleaning up the environment variables

### ISSUE THREAD

**Issue by Isty001** [clibs/clib#219]
I was looking at what kind of env variables are used by the clib commands, and I found two that could use some improvements.

**PREFIX and CLIB_PREFIX**

This piece of code is present at multiple places

```c
  if (opts.prefix) {
    setenv("CLIB_PREFIX", opts.prefix, 1);
    setenv("PREFIX", opts.prefix, 1);
  }
```
But as far as I can tell we don't use the `CLIB_PREFIX` variable anywhere. The `PREFIX` is fetched at two places in `clib-package.c`. I think all that logic should be kept in `clib-package.c`, because `clib_package_install` takes care of that, and that piece of code could be safely removed.

```c
#ifdef CLIB_PACKAGE_PREFIX
  if (0 == opts.prefix) {
#ifdef HAVE_PTHREADS
    pthread_mutex_lock(&lock.mutex);
#endif
    opts.prefix = CLIB_PACKAGE_PREFIX;
#ifdef HAVE_PTHREADS
    pthread_mutex_unlock(&lock.mutex);
#endif
  }
#endif

  if (0 == opts.prefix) {
#ifdef HAVE_PTHREADS
    pthread_mutex_lock(&lock.mutex);
#endif
#ifdef _GNU_SOURCE
    char *prefix = secure_getenv("PREFIX");
#else
    char *prefix = getenv("PREFIX");
#endif

    if (prefix) {
      opts.prefix = prefix;
    }
```

**CLIB_FORCE**

This is always set to but never actually used. If there are no plans to use it, then we should remove it.

```c
  setenv("CLIB_FORCE", "1", 1);
```

/cc @jwerle @stephenmathieson

**Comment by jwerle** [comment]
:+1: Yeah that can be cleaned up and consolidated into `clib-package.c` and yeah :+1: for removing `CLIB_FORCE`

**Comment by Isty001** [comment]
Resolved in #226
